js/repeater-list.js

```diff
@@ -379,7 +379,7 @@
 			this.viewOptions.list_rowRendered({
 				container: $tbody,
 				item: $row,
-				rowData: data
+				rowData: data.items[index]
```

Comment:
These docs will change soon, but: http://getfuelux.com/extensions.html#writing-extensions-repeater-helpers

It's the "subset" of information that is being iterated over. In this case, data.items. `renderRow` is called passing the `helpers.subset` value as the `subset` argument. I suppose it could be renamed "items" in the scope of the `renderRow` function...